lib/hammer_cli_katello/filter.rb

```diff
@@ -53,6 +53,13 @@ class CreateCommand < HammerCLIKatello::CreateCommand
       success_message _("Filter created")
       failure_message _("Could not create the filter")
 
+      validate_options do
+        organization_options = [:option_organization_id, :option_organization_name, \
+                                :option_organization_label]
+
+        any(*organization_options).required
+      end
```

Comment:
You could use `validate_options` - i.e. https://github.com/Katello/hammer-cli-katello/blob/364397877e80cf99d098748bb418476f1fea079a/lib/hammer_cli_katello/host_package_group.rb#L12